pqdb-frontend/src/components/views.js

```diff
@@ -68,30 +288,32 @@ class CustomSQLQuery extends React.Component {
                                         variant="outlined"
                                         error={this.state.error != null}
                                         helperText={this.state.error ? this.state.error.toString() : ""}
-                                        value={this.state.sqlQuery}
-                                        onChange={e => this.setState({ sqlQuery: e.target.value })}
+                                        value={this.state.sqlInput}
+                                        onChange={e => this.setState({ sqlInput: e.target.value })}
                                     />
```

Comment:
We can remove the 'rows' property, then it will autosize.

Reply:
Or use 'rowsMax'

Reply:
autosize depending on the content? sounds good. But I'd still allow manual resizing because why not let users work the way they want. (And I'd not set a maximum height. Let users do what they want)

basically, like this field here: grow when you enter more, but allow manual resizing. (They seem to stop autoresizing at about 80% of the screen height. That's sensible probably, but probably not worth the effort)

Reply:
Now it just resizes to fit. That's enough I'd say.

Btw I think I found out how to use style. You need to pass a js object, and use the JS camelCase style names, like so:
`style={{ backgroundColor: 'red' }}`

Reply:
Yes, that's how it works. However, the material components ignore the style property, because they use their own styling system.